takwimu/takwimu_ui/src/components/WhatCanYouDo.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+
+import { Typography, Grid, withStyles } from '@material-ui/core';
+import { PropTypes } from 'prop-types';
+import Section from './Section';
+
+import reasearchIcon from './assets/a-chart.svg';
+import downloadIcon from './assets/cloud-download-93.svg';
+import presentIcon from './assets/computer-upload.svg';
+
+const styles = theme => ({
+  container: {
+    backgroundColor: theme.palette.iceBlue
+  },
+  box: {
+    width: '19rem',
+    height: '17.5rem',
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'space-between',
+    alignItems: 'flex-start',
+    padding: '1.25rem'
+  },
+  label: {
+    fontWeight: 'bold'
+  }
+});
+
+function WhatCanYouDo({ classes }) {
+  return (
+    <Section title="What can you do with Takwimu" titleVariant="h1">
```

Comment:
1. The component spacing is a bit off: Design vs Implementation
![Screenshot from 2019-03-25 09-29-11](https://user-images.githubusercontent.com/1779590/54899456-ef8ac280-4ee0-11e9-974e-7446642509bc.png)
![Screenshot from 2019-03-25 09-29-24](https://user-images.githubusercontent.com/1779590/54899465-f3b6e000-4ee0-11e9-94b5-6da38cf8d499.png)
2. Breaks on mobile
![Screenshot from 2019-03-25 09-28-50](https://user-images.githubusercontent.com/1779590/54899488-03362900-4ee1-11e9-8296-a434214aad5b.png)